cvat/apps/engine/serializers.py

```diff
@@ -1207,6 +1206,65 @@ class SubLabeledShapeSerializer(ShapeSerializer, AnnotationSerializer):
 class LabeledShapeSerializer(SubLabeledShapeSerializer):
     elements = SubLabeledShapeSerializer(many=True, required=False)
 
+class LabeledImageSerializerFromDB(serializers.BaseSerializer):
```

Comment:
Do we need this to be a serializer at all? Given the usage, it seems like it would be simpler to just use a plain function.

Reply:
Why is it better? Can you summarize the discussion?

Reply:
Because it is in general serialization logic, and we keep it in the serializer. 
But very basic, which works quick:

 https://www.django-rest-framework.org/api-guide/serializers/#baseserializer